src/types.ts

```diff
@@ -32,8 +33,20 @@ export interface ZAuctionInstances {
   [registrarAddress: string]: zAuction.Instance;
 }
 
-export interface RouteUriToInstance {
-  [key: string]: ZAuctionInstances;
+export interface TokenAddressMapping {
+  [address: string]: Mapping;
+}
+
+export interface Mapping {
+  [address: string]: {
+    id: string,
+    name: string
+  };
+}
+
+export interface TokenInfo {
```

Comment:
Nit: too generic of a name. What information is this? What's it's specific usage?
Seems like this is pricing information so maybe `TokenPricingInfo`